sdcm/sct_runner.py

```diff
@@ -667,6 +667,11 @@ def _create_instance(self,
                 "block-project-ssh-keys": "true",
                 "ssh-keys": f"{self.LOGIN_USER}:{self.key_pair.public_key.decode()}",
             },
+            # NOTE: following is needed for the possibility to use the K8S/GKE APIs
+            ex_service_accounts=[{
```

Comment:
Do we understand how it was working in the old project ? The default service had it enabled somehow ?